update

bryant-smith opened the pull request.

quadrantsec commented:
Not sure if I like disabling "Windows audit log was cleared".   I've seen to many attackers clear/alter the logs and its a good TTP.

quadrantsec commented:
If its "noisy" ,  wouldn't the be better to auto answer?

bryant-smith commented:
It was only because it was a duplicate rule.  We had it in windows-misc and one in windows-security.  I removed the windows-misc and updated the windows-security to include eventid 517
